src/index.js

```diff
@@ -104,7 +108,7 @@ export const style = ({
           styles.push({ [media]: rule })
         }
       }
-      styles.sort()
+      if (!isIE) styles.sort()
```

Comment:
Thanks! I think it'd be better to not add an additional dependency and do browser checks for this sort of thing. I assume there's a way to handle this sort in a way that should work cross-browser